### PR TITLE
fix(video) squelch misleading error

### PR DESCRIPTION
**Pull request by saghul**
On Safari it's "normal" to get AbortError. Since there is nothing to do here,
silence the error instead of polluting the logs and making finding other issues
more complicated.

### CODE REVIEW

**Comment by jallamsetty1** [comment]
@saghul We shouldn't be seeing this error in Safari now since we are adding the audio track to the pc always (muting after if the user joins muted). IIRC Safari throws this exception on play only when there is no local media capture happening. Does the video play irrespective of the error ?

**Comment by saghul** [comment]
I see this all the time when joining from Safari.

**Comment by saghul** [comment]
Oh and on video tracks FWIW.

**Comment by saghul** [comment]
Well, actually hold on. Unless I click looks like the prejoin screen gets black video, once I click anywhere it stats playing.

File: react/features/base/media/components/web/Video.js [hunk 1]
```diff
@@ -203,14 +203,7 @@ class Video extends Component<Props> {
...
             // case video does not autoplay.
-            this._videoElement.play()
-                .catch(error => {
-                    // Prevent uncaught "DOMException: The play() request was interrupted by a new load request"
-                    // when video playback takes long to start and it starts after the component was unmounted.
-                    if (this._mounted) {
-                        throw error;
-                    }
-                });
+            this._videoElement.play().then(null, noop);
```
Comment (paweldomas) [inline react/features/base/media/components/web/Video.js:206]: I think it's dangerous to just swallow the errors. How about filtering by browser type and maybe trying to match the specific error that is ignored instead?

**Comment by jallamsetty1** [comment]
> Well, actually hold on. Unless I click looks like the prejoin screen gets black video, once I click anywhere it stats playing.

https://bugs.webkit.org/show_bug.cgi?id=183333#c14
Looks like capture/user gesture is required for audio to play, not sure if its the same for video as well.

**Comment by saghul** [comment]
Let's drop this.

**Pull request closed without merging**